modules/tofi/hm.nix

```diff
@@ -1,34 +1,35 @@
-{
-  config,
-  lib,
-  ...
+{ config
+, lib
+, ...
 }:
 with config.stylix.fonts;
 with config.lib.stylix.colors.withHashtag; {
   options.stylix.targets.tofi.enable =
     config.lib.stylix.mkEnableTarget "Tofi" true;
 
   config = lib.mkIf config.stylix.targets.tofi.enable {
-    programs.tofi.settings = let
-      background = base00;
-      foreground = base05;
-      darkForeground = base04;
-      selection = base03;
-    in {
-      font = monospace.name;
-      font-size = toString sizes.popups;
-
-      background-color = background;
-      outline-color = darkForeground;
-      border-color = foreground;
-      text-color = foreground;
-      prompt-color = base0A;
-      prompt-background = background;
-      placeholder-color = selection;
-      input-background = background;
-      default-result-background = background;
-      selection-color = selection;
-      selection-background = background;
-    };
+    programs.tofi.settings =
+      let
+        opacity = lib.toHexString ((((builtins.ceil (config.stylix.opacity.popups * 100)) * 255) / 100));
+        background = base00 + opacity;
+        foreground = base05 + opacity;
+        darkForeground = base04 + opacity;
+        selection = base03 + opacity;
+      in
+      {
+        font = monospace.name;
+        font-size = toString sizes.popups;
+        background-color = background;
+        outline-color = darkForeground;
+        border-color = foreground;
```

Comment:
With the current state of opacity support, borders tend to remain opaque. Perhaps that should also be the case here? I'm open to suggestions either way.

Reply:
I would agree with reducing the border width. We don't currently have an option to define a standard width, though. Either way, that can be worked on in a future PR to avoid holding this one up for too long.